apps/meteor/app/api/server/v1/users.ts

```diff
@@ -226,6 +227,43 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'users.setAvatarFromService',
```

Comment:
there is no need to create a new endpoint.. we already have the endpoint `users.setAvatar`, if it is missing this functionality we should just improve it.